File: app/src/main/java/org/kindone/willingtodo/data/Task.java
package org.kindone.willingtodo.data;

/**
 * Created by kindone on 2015. 11. 7..
 */
public class Task {
    public final long id;
    public final String title;
    public final long contextId;
    public final String category;
    public final String deadline;
    public final int priority;
    public final int willingness;

    public Task(long id, String title, long contextId, String category, String deadline, int priority, int willingness) {
        this.id = id;
        this.title = title;
        this.contextId = contextId;
        this.category = category;
        this.deadline = deadline;
        this.priority = priority;
        this.willingness = willingness;
    }

    public String toString() {
        return "id = " + id +
                ", title = " + title +
                ", contextId = " + contextId +
                ", category = " + category +
                ", deadline = " + deadline +
                ", priority = " + priority +
                ", willingness = " + willingness;
    }
}

File: app/src/main/java/org/kindone/willingtodo/data/Task.kt
package org.kindone.willingtodo.data

/**
 * Created by kindone on 2015. 11. 7..
 */
class Task(val id: Long, val title: String,
           val contextId: Long, val category: String,
           val deadline: String, val priority: Int, val willingness: Int) {

    constructor(title: String, contextId: Long) : this(0, title, contextId, "", "", 0, 0) {}

    override fun toString(): String {
        return "id = " + id +
                ", title = " + title +
                ", contextId = " + contextId +
                ", category = " + category +
                ", deadline = " + deadline +
                ", priority = " + priority +
                ", willingness = " + willingness
    }
}
